README.md

```diff
@@ -80,6 +84,45 @@ Terminal 3: Read from GET interface
 curl http://localhost:60023/log/secretpuppy
 ```
 
+### Example 2 - secure over https, using non-default ports
+
+For this example, **you'll need a valid SSL key and certificate** for some
+domain.
+
+Let's say, your SSL key/certificate are for domain `taskcluster-worker.net`,
+and you have them on your filesystem at locations `~/myssl.crt` and
+`~/myssl.key` in PEM format (`-----BEGIN` ...).
+
+First make sure a name under *your domain* (in this example, we'll use
+`pete.taskcluster-worker.net`) resolves to your local machine:
+
+```
+echo '127.0.0.1 pete.taskcluster-worker.net' | sudo tee -a /etc/hosts >/dev/null
```

Comment:
That works if you have write access to /etc/hosts, but otherwise will fail.

The reason for using `tee` is because just running:
```
sudo echo ... > /etc/hosts
```
would run the echo command as root, but the redirect would still operate as the current user - so using `tee` is a trick to write inside the process, rather than redirecting the output of the process.